src/components/application_manager/rpc_plugins/sdl_rpc_plugin/src/commands/hmi/on_exit_application_notification.cc

```diff
@@ -112,6 +112,14 @@ void OnExitApplicationNotification::Run() {
       break;
     }
 #endif  // CLOUD_APP_WEBSOCKET_TRANSPORT_SUPPORT
+    case Common_ApplicationExitReason::RESOURCE_CONSTRAINT: {
+      const auto message =
+          MessageHelper::GetOnAppInterfaceUnregisteredNotificationToMobile(
+              app_id, AppInterfaceUnregisteredReason::RESOURCE_CONSTRAINT);
+      SendNotificationToMobile(message);
+      application_manager_.UnregisterApplication(app_id, Result::SUCCESS);
+      return;
```

Comment:
@mvorobio So we don't we need to disconnect WEP ap as a client?

Reply:
@AByzhynar  `final_message` flag is used to specify that a connection should be closed after `OnAppInterfaceUnregisteredNotification` is sent. It is ProtocolHandler's responsibility.